common/scala/src/whisk/core/entity/Exec.scala

```diff
@@ -76,6 +76,10 @@ protected[core] case class JavaExec(jar: String, main: String) extends Exec(Exec
 
 protected[core] case class BlackBoxExec(image: String) extends Exec(Exec.BLACKBOX)
 
+protected[core] case class SequenceExec(code: String, components: Vector[String]) extends Exec(Exec.SEQUENCE) {
```

Comment:
I was going to leave it as a string for this round as the underlying implementation is still with pipe.js.  Next round, Vector[String] becomes Vector[Namespace] and code field disappears.